combat-trainer.lic

```diff
@@ -375,7 +375,10 @@ class LootProcess
       end
     end
     pause 0.25
-    bput("drop #{item}", 'You drop') if Flags['container-full']
+    if Flags['container-full']
+      bput("drop #{item}", 'You drop')
```

Comment:
should this be drop my #{item}"? drop item could default to the one on the ground and fail

Reply:
This shouldn't be na issue, and everything but the adding of the item to unlootables is already in there working fine. The item comes from a list of room objects. So, 99.9999% chance that the same two items don't drop from the same critter, and if it's on the no loot list, it will already need to apply to all.